FHIR-Converter/server/src/common/utils.ts

```diff
@@ -5,16 +5,25 @@
 
 import * as path from 'path';
 import * as glob from 'glob';
-import * as constants from '../common/constants'; 
+import * as constants from '../common/constants';
 
 export function addUnderlineExt(filePath: string) {
-	filePath = path.join(path.dirname(filePath), '_' + path.basename(filePath) + constants.EngineTemplateFileExt);
+	const dirname = path.dirname(filePath);
+	const filenameWithExt = path.join(path.basename(filePath) + constants.EngineTemplateFileExt);
+	if (dirname === '.') {
+		filePath = filenameWithExt;
```

Comment:
Can we add test case for this branch?

Reply:
I have added test case for this branch in untils.test.ts:
```
test('Function addUnderlineExt - should return the fullname of snippet template with extension but without underline', () => {
	const templateName = utils.addUnderlineExt('snippet_template');
	assert.strictEqual(templateName, 'snippet_template.liquid');
});
```

Reply:
Seems test case not covered when dirname === '.'?